### PR TITLE
Added buildscripts for 8 packages

### PR DESCRIPTION
**Pull request by csrividya**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: d/dustjs-linkedin/dustjs_ubi_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+WORK_DIR=`pwd`
+
+REPO_URL=https://github.com/linkedin/dustjs.git
+
+PACKAGE_NAME=dustjs
+
+VERSION=${1:-v3.0.0}
+
+dnf install git wget nodejs -y
```
Comment (seth-priya) [inline d/dustjs-linkedin/dustjs_ubi_8.3.sh:32]: is there a specific reason for using nodejs 10? ideally we should be using v12 which can be installed using nvm or "dnf module enable nodejs:12; dnf install nodejs"

File: d/dustjs-linkedin/dustjs_ubi_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+cd $PACKAGE_NAME
+git checkout ${VERSION}
+
+#Build repo
+npm install yarn -g
+yarn install
+
+#Test repo
+yarn test
+
```
Comment (seth-priya) [inline d/dustjs-linkedin/dustjs_ubi_8.3.sh:52]: Please remove the blank lines at the end of the script

File: f/filehandler/finalhandler_ubi_8.3.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+WORK_DIR=`pwd`
+
+REPO_URL=https://github.com/pillarjs/finalhandler.git
+
+PACKAGE_NAME=finalhandler
+
+VERSION=${1:-v0.4.1}
+
+dnf install git  -y
+wget https://nodejs.org/dist/v7.0.0/node-v7.0.0-linux-ppc64le.tar.gz &&
```
Comment (seth-priya) [inline f/filehandler/finalhandler_ubi_8.3.sh:33]: any particular reason to use tis version?

File: j/jstransformer/jstransformer_ubi_8.3.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+git checkout ${VERSION}
+
+#Build repo
+npm install yarn -g
+yarn install
+
+#Test repo
+yarn test
+yarn run coverage
+
```
Comment (seth-priya) [inline j/jstransformer/jstransformer_ubi_8.3.sh:47]: Please remove blank lines

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: d/dustjs-linkedin/dustjs_ubi_8.3.sh (hunk 1 above)
Comment (csrividya) [reply d/dustjs-linkedin/dustjs_ubi_8.3.sh:32]: yes @seth-priya version 12 also works fine I will update the script and commit

**Review by seth-priya — APPROVED** [review]

**Pull request merged**